docs/version_bump_template.md

```diff
@@ -30,23 +31,21 @@ General components:
 ```
 - k3s
 ```
-1- flannel
-2- coredns
-3- metrics
-4- etcd
-5- plugins
-6- traefik
-7- local-path-storage
-8- containerd
-9- klipper
-10- runc
+1- coredns
+2- metrics server
+3- etcd
+4- cni plugins
+5- traefik
+6- local-path-storage
```

Comment:
local-path-provisioner
metrics-server
We want flannel here. Any reason to remove?

Reply:
flannel we removed for a single job - https://github.com/rancher/distros-test-framework/blob/445041c9c5c1c36a9fb0ea330a5f9991ae7aa885/entrypoint/versionbump/flannelbump_test.go#L35

it runs for k3s and rke2- we are just missing creating a jenkins job for k3s - for rke2 its already there

Reply:
@ShylajaDevadiga